src/domain/constants.js

```diff
@@ -223,3 +225,11 @@ export const DEFAULT_COLOR_SWATCHES = [
   ['#fffbe6', '#fff1b8', '#ffe58f', '#ffd666', '#ffc53d', '#faad14', '#d48806', '#ad6800', '#874d00', '#613400'],
   ['#ffffff', '#fafafa', '#f5f5f5', '#e8e8e8', '#d9d9d9', '#bfbfbf', '#8c8c8c', '#595959', '#262626', '#000000']
 ];
+
+export const HTTP_STATUS = {
+  badRequest: 400,
+  unauthorized: 401,
```

Comment:
I took the official HTTP message wording here which is misleading, should have been "unauthenticated" actually.

Reply:
I remember we talked about this before and you pointed this out 👍  I agree, we should go with the official naming though..